docs/guides/admin/docs/modules/youtubepublication.md

```diff
@@ -13,40 +12,48 @@ You need to meet these requirements to make a YouTube Publication:
 
 ## Google Developers Configuration
 
+Below is a summarized version of Google's quickstart page, which currently lives [here][googledoc].  If these
```

Comment:
Suggestion:

    Below is a summarized version of [Google's quickstart page][googledoc].